tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -150,6 +150,37 @@ describe('Home - Game List', () => {
 
 describe('Home - Create Game', () => {
   beforeEach(setup);
+
+  it.only('Saves ranked setting between sessions', () => {
```

Comment:
```suggestion
  it('Saves ranked setting between sessions', () => {
```

Reply:
Whoops, thank you. Missed that 😓 -- really need a precommit hook to catch those, haha.